### ISSUE TITLE
Just Found: gulp with Error in plugin 'gulp-cache'

### ISSUE THREAD

**Issue by zhaiduo** [taigaio/taiga-front#1416]
just Merge master with Commits on Mar 2, 2018 https://github.com/taigaio/taiga-front/commit/9a6541f6e0ef49097c0f799e42ccd02f8763d699

then run gulp with the following error:
> [17:19:49] Starting 'scss-lint'...
Unhandled rejection Error in plugin 'gulp-cache'
Message:
    Error code 1
[object Object] 

It seems the problem with scss-lint.

When I apply cherry-pick, error comes with the following:
> Error in plugin 'gulp-scss-lint'
Message:
    ScssLint failed for: styles/modules/kanban/kanban-table.scss
pre-commit:
pre-commit: We've failed to pass the specified git pre-commit hooks as the `scss-lint`
pre-commit: hook returned an exit code (1). If you're feeling adventurous you can
pre-commit: skip the git pre-commit hooks by adding the following flags to your commit:
pre-commit:
pre-commit:   git commit -n (or --no-verify)
pre-commit:
pre-commit: This is ill-advised since the commit is broken.
pre-commit:

I'm not good at scss Problem, wish this can help it out.

**Comment by zhaiduo** [comment]
with this commit https://github.com/taigaio/taiga-front/commit/ebe70b2f0c9adaf1593a506fe1ed6d9aaa4dc1ed, there's no problem with gulp start.

**Comment by migonzalvar** [comment]
You're right! I think it will be solved with #1417

**Comment by jwir3** [comment]
I seem to be getting this error once more on the current `HEAD` of `master` when building with Ubuntu 18.04.1 LTS.

**Comment by jwir3** [comment]
Ugh. So I think I found the issue. I had originally tried to build `taiga-front` with a local user, `taiga`. I then went back and wanted to edit the source using my normal user, `scottj`. Apparently `gulp-cache` creates a directory called `/tmp/gulp-cache` which I couldn't delete (since it was created with the original `taiga` user and not `scottj`).

I diagnosed this problem by running `gulp deploy`, which wouldn't run indefinitely, and reported errors on the command line more verbosely. 

The workaround is to go into `/tmp/` and manually delete the `gulp-cache` directory using: `cd /tmp && sudo rm -rf gulp-cache`. I'll see if I can submit a patch that uses a better temp directory name.
